components/form.html

```diff
@@ -5,31 +5,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
     <link rel="stylesheet" href="./styles.css" />
+    
+    <script type="module" src="./form.js" defer></script>
```

Comment:
If you work with `type="module"` you don't need `defer`.

```suggestion
    <script type="module" src="./form.js"></script>
```